internal/worldcomm/worldcomm.go

```diff
@@ -80,25 +73,25 @@ type WorldCommunicationState struct {
 	webRtc     webrtc.IWebRtc
 	Reporter   IReporter
 
-	authMethod         string
-	coordinator        *Coordinator
-	Peers              map[string]*peer
-	peersIndex         PeersIndex
-	topicQueue         chan *topicChange
-	connectQueue       chan string
-	webRtcControlQueue chan *protocol.WebRtcMessage
-	messagesQueue      chan *peerMessage
-	unregisterQueue    chan string
-	aliasChannel       chan string
-	stop               chan bool
-	softStop           bool
-	ReportPeriod       time.Duration
+	authMethod            string
+	coordinator           *Coordinator
+	Peers                 map[string]*peer
```

Comment:
I think we should be able to keep a []*peer instead and use pointers instead of alias

Reply:
a) subscriptions field is actually a subset of this information, let's remove one
b) we don't need peer information, just a topic count. Since the peer knows their own topics (and then, his state on this structure), we should be able to update the count without addiotional information